src/scss/06_components/forms/_react-forms.scss

```diff
@@ -0,0 +1,83 @@
+// Checkbox styles need specific selectors to override HDS styles
+.hdbt-search__filter.hdbt-search__checkbox {
+  --background-unselected: transparent;
+  --background-selected: var(--hdbt-color-black);
+  --background-hover: var(--hdbt-color-black);
+  --border-color-selected: var(--hdbt-color-black);
+  --border-color-selected-hover: var(--hdbt-color-black);
+  --border-color-selected-focus: var(--hdbt-color-black);
+  --border-color-unselected: var(--hdbt-color-black);
+  --border-color-unselected-hover: var(--hdbt-color-black);
+  --border-color-unselected-focus: var(--hdbt-color-black);
+  --icon-color-unselected: transparent;
+  --icon-color-selected: var(--hdbt-text-color);
+  --icon-color-disabled: var(--hdbt-text-color);
+  --focus-outline-color: var(--hdbt-color-black);
+}
+
+.hdbt-search__filter.hdbt-search__checkbox label {
+  font-weight: $font-weight-regular;
+}
+
+.hdbt-search__filter {
+  margin-bottom: $spacing;
+}
+
+// Override HDBTs li+li
+.hdbt-search__filter li {
+  margin-top: 0;
+}
+
+.hdbt-search__filter label {
+  color: $color-black;
+  display: block;
+  font-weight: $font-weight-medium;
+  margin-bottom: $spacing-quarter;
+}
+
+// Need specific selector here to override HDS styles
+.hdbt-search__submit-button:not(:disabled) {
+  --background-color: var(--hdbt-color-black);
+  --background-color-hover: var(--hdbt-text-color);
+  --background-color-focus: var(--hdbt-text-color);
+  --background-color-hover-focus: var(--hdbt-text-color);
+  --border-color: var(--hdbt-color-black);
+  --border-color-hover: var(--hdbt-color-black);
+  --border-color-focus: var(--hdbt-color-black);
+  --border-color-hover-focus: var(--hdbt-color-black);
+  --color: var(--hdbt-text-color);
+  --color-hover: var(--hdbt-color-black);
+  --color-focus: var(--hdbt-color-black);
+  --color-hover-focus: var(--hdbt-color-black);
+  --focus-outline-color: var(--hdbt-color-black);
+}
+
+// Override HDS styles for date input
+.hdbt-search__filter.hdbt-search__date-input {
+  --focus-outline-color: var(--hdbt-color-black);
+}
+
+// Override HDBT table styles to fix date picker
+.hdbt-search__date-input table {
+  border: none;
+  display: table;
+  margin-top: 0;
+}
+
+.hdbt-search__date-input table tbody tr td {
+  min-width: unset;
+}
+
+.hdbt-search__date-input table thead th {
+  background: $color-white;
+  color: $color-black;
+  padding-left: 0;
+  padding-right: 0;
+  padding-top: 0;
+  text-align: center;
+}
+
+.hdbt-search__date-input table tbody tr td,
+.hdbt-search__date-input table thead th {
+  min-width: unset;
```

Comment:
Seems it's not needed so I'll remove it. Maybe HDS has made some updates since this was added. It's not my code so I don't know more.

Reply:
Could you please link the ticket here both in ticket and in code so that we remember to fix it?

Reply:
I removed the min-width from here because seems it's not needed but I can add comment about the ticket if you want to check this still when working with the ticket.

Reply:
seems I forgot to push code yesterday evening but as said min-width is now removed but added also comment about checking it with 3044.